bin/api_diff.ml

```diff
@@ -38,6 +54,13 @@ let info =
   Cmd.info "api-watcher" ~version:"%%VERSION%%" ~exits:Cmd.Exit.defaults
     ~doc:"List API changes between two versions of a library"
 
-let term = Cmdliner.Term.(const run $ ref_cmi $ current_cmi)
+let run_with_error_handling ref_cmi current_cmi =
+  match run ref_cmi current_cmi with
+  | Ok code -> code
+  | Error msg ->
+      Printf.eprintf "Error: %s\n" msg;
+      2
```

Comment:
You can use this `Cmdliner` function in place of `eval`: https://erratique.ch/software/cmdliner/doc/Cmdliner/Cmd/index.html#val-eval_result'

That way you can build the cmdliner term with `run`.